spec/controllers/users_controller_spec.rb

```diff
@@ -11,7 +11,11 @@
   describe "POST create" do
     context "sending emails" do
       after {ActionMailer::Base.deliveries.clear}
-      before {ActionMailer::Base.deliveries.clear}
+      let(:charge) {double(:charge, successful?: true)}
+      before do
+        ActionMailer::Base.deliveries.clear
+        StripeWrapper::Charge.stub(:create).and_return(charge)
```

Comment:
Ah, that means the method is actually not being called. I should have shown you both syntaxes.

```ruby
expect(StripeWrapper::Charge).to receive(:create).and_return(charge)
```

and

```ruby
allow(StripeWrapper::Charge).to receive(:create).and_return(charge)
```

The first will cause the spec to fail if there is no call to the method. This should be used when the call to the method is absolutely required for your code base to be functioning properly. The second is when you don't necessarily care if the method is called, but if it is you need it to be stubbed.